docs/05-technical-reference/00-architecture/ac-03-application-gateway.md

```diff
@@ -13,7 +13,7 @@ which are either unsecured or secured with various security mechanisms and prote
 
 1. A Function calls Application Gateway. 
 2. Application Gateway extracts the Application name and the service name from the path. Using the extracted Application name, Application Gateway finds the respective Application custom resource and obtains the information about the registered external API, such as the API URL and security credentials. 
-3. Application Gateway gets a token from the OAuth server.
+3. Application Gateway gets a token from the OAuth server. The OAuth server can be additionally secured with client TLS certificate.
```

Comment:
```suggestion
3. Application Gateway gets a token from the OAuth server. The OAuth server can be additionally secured with a client TLS certificate.
```
We could also link here to the tutorial on registering a secured API. 

```
3. Application Gateway gets a token from the OAuth server. The OAuth server can be additionally [secured with a client TLS certificate](../../03-tutorials/00-application-connectivity/ac-04-register-secured-api.md#register-an-o-auth-secured-api-with-token-obtained-from-client-certificate-secured-endpoint).
```
(The link must be checked if used.)

Reply:
Please go ahead with the change